src/components/transactions/message-view/WithdrawDelegationRewardMessageDetails.vue

```diff
@@ -83,10 +81,10 @@
 </template>
 
 <script>
-import { atoms, viewDenom, prettyLong } from "scripts/num.js"
-import { resolveValidatorName } from "src/filters"
-import TransactionIcon from "../TransactionIcon"
-import Avatar from "common/Avatar"
+import { atoms, viewDenom, prettyLong } from 'scripts/num.js'
+import { resolveValidatorName } from 'src/filters'
+import TransactionIcon from '../TransactionIcon'
+import Avatar from 'common/Avatar'
```

Comment:
Why is the linter changing now all this?

Reply:
good question o.O